src/components/Dashboard/MainInput/MainInput.jsx

```diff
@@ -54,27 +36,22 @@ export default class MainInput extends React.Component {
     });
   };
 
-  setInputValue = (value) => {
+  setInputValue = (value = '') => {
     const { newPriority } = this.state;
-    const inputDataArr = value.split('');
+    const inputDataArr = value;
     const hashCheck = inputDataArr[inputDataArr.length - 2];
     let setPriority = newPriority;
```

Comment:
will be overrided inside changePriority ?

Reply:
Yes, it was stupid. I changed it.